odoo_module_migrate/migration_scripts/migrate_150_160.py

```diff
@@ -9,6 +9,17 @@
     },
 }
 
+_RENAMED_MODELS = [
+    ("stock.production.lot", "stock.lot", None),
```

Comment:
Maybe you should put renamed models into separate files? Similar to `file_renames` https://github.com/OCA/odoo-module-migrator/blob/master/DEVELOP.rst

This way we avoid merge conflicts in PR. More info: https://github.com/OCA/odoo-module-migrator/pull/42

Anyway, please update DEVELOP.rst about the new feature

Reply:
Hi @yelizariev 

I updated the PR with your suggestions... I wasn't aware of the new logic of files and folders, but I think it is a good improvement. Just a comment, it seems weird that I had to create the file "migrate_150_160.py" anyway with the empty class... probably we could improve this in the future.